src/components/modules/Authentcation/Changepassword/ChangePassword.tsx

```diff
@@ -1,5 +1,89 @@
 import React from "react";
+import { useNavigate } from "react-router-dom";
+import { ChangePass } from "../../../../../interface/Interface";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { AUTHURL } from "../../../../constans/EndPoints/ENDPOINT";
+import { toast } from "react-toastify";
+import { VALIDATIONS } from "../../../../constans/Validation/Validation";
 
 export default function ChangePassword() {
-  return <div>ChangePassword</div>;
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<ChangePass>();
+  const onsubmit = async (data: ChangePass) => {
+    try {
+      const response = await axios.post(AUTHURL.CHANGE, data);
+      console.log(response.data);
+      toast.success(response.data.message);
+      navigate("/Login");
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  return (
+    <>
+      <h1 className="py-5 font-bold text-3xl text-auttextcolor">
+        Change password
+      </h1>
+      <div className="py-10">
+        <form onSubmit={handleSubmit(onsubmit)}>
+          <label htmlFor="email" className="block py-3">

```

Comment:
We can make a reusable input component instead of repeating the tailwind class and the structure for each one. The current implementation will make it difficult to update code.